docs/swiperefresh.md

```diff
@@ -34,7 +47,34 @@ SwipeRefresh(
 }
 ```
 
-The full example, including the view model implementation can be found [here](https://github.com/google/accompanist/blob/main/sample/src/main/java/com/google/accompanist/sample/swiperefresh/DocsSamples.kt).
+The full example, including the view model implementation can be found [here](https://github.com/google/accompanist/blob/main/sample/src/main/java/com/google/accompanist/sample/swiperefresh/docsamples/SimpleSample.kt).
+
+Alternatively, you can use a bottom refresh indicator or even both a top and bottom indicator (e.g: If you are loading the list items by page, each time the user triggers the bottom refresh indicator, you load the next page. If the user triggers the top refresh indicator, you refresh the list):
+
+``` kotlin
+val viewModel = viewModel<SampleViewModel>()
+val isTopIndicatorRefreshing by viewModel.isRefreshing.collectAsState()
+val isBottomIndicatorRefreshing by viewModel.isLoadingNextPage.collectAsState()
+    
+SwipeRefresh(
+    topRefreshIndicatorState = rememberSwipeRefreshState(isTopIndicatorRefreshing),
```

Comment:
A layout should only have 1 state instance which should cover the entire state of the layout. The API for this should look something like imo:

``` kotlin
enum class Position { Top, Bottom }

SwipeRefresh(
    state: SwipeRefreshState,
    position: Position,
)
```

If someone wants to have both top & bottom refreshes (which isn't a great UX anyway), they can nest two `SwipeRefresh()`s.

Reply:
No problem, that will also simplify the implementation and remove all null checks for the indicator state.

Reply:
Isn't it redundant to have both `position` and `indicatorAlignment` ? In addition, a user can have something like this:
```kotlin
SwipeRefresh(
    ...
    position = Position.Bottom,
    indicatorAlignment = Alignment.TopCenter, // Which is the default.
)
```
Which will cause an unexpected behavior.
We should remove the`indicatorAlignment` or replace its type by `Alignment.Horizontal`.

Reply:
Yeah you're right. We could create a `SwipeRefreshIndicator.Alignment` which == `Alignment` minus any vertical centering. We could then update `indicatorAlignment` to use that alignment.

WDYT?

Reply:
Yeah, it's a good idea. But it will be simpler if we use the built-in [Alignment.Horizontal](https://developer.android.com/reference/kotlin/androidx/compose/ui/Alignment.Horizontal) interface as the `indicatorAlignment`'s type. 
The `Alignment.Horizontal` allow only horizontal alignment, any vertical alignment will cause a compile time error:

![Screen Shot 2021-07-13 at 8 37 44 AM](https://user-images.githubusercontent.com/62622509/125453011-1c4e593e-a5a8-44b9-9649-ea58c26d98e6.png)